packages/application-generic/src/custom-providers/index.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  StandardQueueServiceHealthIndicator,
+  WebSocketsQueueServiceHealthIndicator,
+  WorkflowQueueServiceHealthIndicator,
+} from '../health';
+import {
```

Comment:
Reusable (and often used) custom providers stored in App Generic to share as needed. Gathering them in different modules would help with more domain oriented code.